docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Managing tracking scenarios in the console"
+sidebar_label: "🆕 Managing tracking scenarios in the console"
+sidebar_position: 80
+sidebar_custom_props:
+  offerings:
+    - enterprise
```

Comment:
This is only needed in the parent page. It’s propagated to all child pages automatically

Reply:
Done